src/containers/Userlist.jsx

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import { Select, MenuItem, Alert, AlertTitle } from '@mui/material';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { defaultProfilePic, lock, unlock, loading } from '../assets';
+import {
+  disableUser,
+  enableUser,
+  reset
+} from '../states/features/users/usersSlice';
+import Loading from '../components/Loading';
+
+export const Userlist = ({ userList }) => {
+  const dispatch = useDispatch();
+  const { isLoading, accountStatus, message } = useSelector((state) => {
+    return state.users;
+  });
+  const [open, setOpen] = React.useState(false);
+
+  const [role, setrole] = useState(2);
+  
+  const handleChange = (event) => {
+    setrole(event.target.value);
+  };
+
+  const blockUnblockUser = (e) => {
+    console.log(e.target.id);
+    const handleClose = (reason) => {
+      if (reason === 'clickaway') {
+        return;
+      }
+      setOpen(false);
+    };
+
+    if (accountStatus) {
+      dispatch(disableUser(e.target.id));
+      console.log(message);
+      e.target.src = lock;
+    } else {
+      e.target.src = unlock;
+      dispatch(enableUser(e.target.id));
+    }
+  };
+  return (
+    <div>
+      {message && (
+                <Snackbar
+                open={open}
+                anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+                autoHideDuration={9000}
+                onClose={handleClose}
+              >
+                <Alert
+                  variant='standard'
+                  severity='error'
+                  style={{ fontSize: '20px', padding: '6px 8px' }}
+                >
+                  {message}
+                </Alert>
+              </Snackbar>
+      )}
+      {isLoading && <Loading />}
+      {userList?.map((user) => (
+        /* eslint-disable-next-line */
```

Comment:
remove this eslint comment. If throws errors try to see how you can resolve it not disabling the rules.